src/styles/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: all .3s;
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  &__card:hover {
+    transform: scale(1.2);
+    color: #34568b;
+  }
 }
```

Comment:
Use nesting here more details in  [video](https://mate.academy/learn/css-advanced/sass#/video/925)